src/theme/styled.ts

```diff
@@ -1,3 +1,6 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
You can omit the year from here,  just have this "// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved."

Reply:
Done